scripts/installers/install-minikube.sh

```diff
@@ -10,44 +10,53 @@ export CHANGE_MINIKUBE_NONE_USER=true
 export KUBECONFIG=$HOME/.kube/config
 
 function main () {
-  echo "****** Installing minikube..."
   install_minikube
 
   echo "****** Verifying installation..."
   kubectl cluster-info
   wait_for_kube
-  ## Run tests below
-  echo "Minikube enabled job is running..."
+  
+  echo "****** Minikube enabled job is running..."
 
 }
 
 function install_minikube() {
+  echo "****** Installing minikube for linux..."
+
   sudo apt-get update -y
   sudo apt-get -qq -y install conntrack
+
   ## get kubectl
-  curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.18.1/bin/linux/amd64/kubectl \
-      && chmod +x kubectl \
-      && sudo mv kubectl /usr/local/bin/
+  echo "****** Installing kubectl v1.20.1..."
+  curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.20.1/bin/linux/amd64/kubectl \
+  && chmod +x kubectl \
+  && sudo mv kubectl /usr/local/bin/ 
+
   ## Download minikube
-  curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.8.1/minikube-linux-amd64 \
-      && chmod +x minikube \
-      && sudo mv minikube /usr/local/bin/
+  echo "****** Installing Minikube v1.21.0..."
+  curl -Lo minikube https://storage.googleapis.com/minikube/releases/v1.21.0/minikube-linux-amd64 \
+  && chmod +x minikube \
+  && sudo mv minikube /usr/local/bin/
+
 
   mkdir -p $HOME/.kube $HOME/.minikube
   touch $KUBECONFIG
-  sudo minikube start --profile=minikube --vm-driver=none --kubernetes-version=v1.18.1
+  minikube start --profile=minikube --kubernetes-version=v1.20.1
```

Comment:
Let's use kubectl 1.19 since OpenShift 4.6 (Long supported version) is at that level.